contracts/modules/dispute/DisputeModule.sol

```diff
@@ -239,19 +260,69 @@ contract DisputeModule is
         emit DisputeCancelled(disputeId, data);
     }
 
+    /// @notice Tags a derivative if a parent has been tagged with an infringement tag
+    /// @param parentIpId The infringing parent ipId
+    /// @param derivativeIpId The derivative ipId
+    /// @param parentDisputeId The dispute id that tagged the parent ipId as infringing
+    function tagDerivativeIfParentInfringed(
```

Comment:
Who is supposed to execute this function?

Reply:
it's permissionless